test/e2e/lib/pages/gutenboarding/acquire-intent-page.js

```diff
@@ -17,7 +17,7 @@ export default class AcquireIntentPage extends AsyncBaseContainer {
 
 	async enterSiteTitle( siteTitle ) {
 		const siteTitleSelector = By.css( '.acquire-intent-text-input__input' );
-		return await driverHelper.setWhenSettable( this.driver, siteTitleSelector, siteTitle );
+		return await driverHelper.setWhenSettable( this.driver, siteTitleSelector, siteTitle, { pauseBetweenKeysMS: 10 } );
```

Comment:
Adding this pause makes the first part of the `Gutenboarding: Create new site as existing user @parallel @canary` test pass. There was a race condition that would keep it from entering the text properly